test/runtests.jl

```diff
@@ -107,4 +107,20 @@ cov_types = (HermOrSym, Cholesky)
             bayes_rule_test(T, n, m, cov_types, matrix_types)
         end
     end
+
+    @testset "Code quality (Aqua.jl)" begin
+        Aqua.test_all(
+            MarkovKernels,
+            ambiguities = false,
+            unbound_args = false,
+            piracies = false,
```

Comment:
@filtron I had to specify these three keyword arguments as there are some issues otherwise. Should we fix these here, or just open an issue and do this later?

Reply:
We can do it later.